test/pylib/s3_proxy.py

```diff
@@ -216,7 +217,12 @@ def do_HEAD(self):
 # addition, it is possible just to start this server using another script - `start_s3_proxy.py` to run it locally to
 # provide proxy between tests and minio
 class S3ProxyServer:
-    def __init__(self, host: str, port: int, minio_uri: str, max_retries: int, logger=None):
+    def __init__(self, host: str, port: int, minio_uri: str, max_retries: int, seed: int = None, logger=None):
+        if seed is None:
+            with open("/dev/urandom", "rb") as f:
+                seed = int.from_bytes(f.read(8), "big")
+        print(f'Setting minio proxy random seed to {seed}')
+        random.seed(seed)
```

Comment:
IMHO, no need to set seed here. the caller can always set it before running `S3ProxyServer`.

Reply:
I dont see how it makes any difference in this particular case but on the other hand I dont mind, so will move the seed initialization to the `test.py`